plugins/generic/externalFeed/simplepie/SimplePie.inc.php

```diff
@@ -1,15009 +0,0 @@
-<?php
-/**
- * SimplePie
```

Comment:
I forget if we discussed this elsewhere -- but is SimplePie available as a git submodule or (better yet) Composer dependency? We should avoid adding third-party code with straight-up `git add`.